lib/topology/platforms/shell.py

```diff
@@ -618,6 +618,7 @@ def connect(self, *args, connection=None, **kwargs):
                 ]),
                 category='connection'
             )
+        spawn.setwinsize(30, 150)
```

Comment:
Please explain why is this necessary.

Reply:
This helps minimize the amount of line changes inserted by the pseudo-tty pexpect uses

Reply:
Want me to comment it in code?